odfe-cli/it/helper.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package it
+
+import (
+	"bytes"
+	"context"
+	"es-cli/odfe-cli/client"
+	"es-cli/odfe-cli/entity"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/stretchr/testify/suite"
+)
+
+const EcommerceIndexName = "integration-test-ecommerce"
+
+type ODFECLISuite struct {
+	suite.Suite
+	Client  *client.Client
+	Profile *entity.Profile
+}
+
+//HelperLoadBytes loads file from testdata and stream contents
+func HelperLoadBytes(name string) []byte {
+	path := filepath.Join("testdata", name) // relative path
+	contents, err := ioutil.ReadFile(path)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	return contents
+}
+
+// DeleteEcommerceIndex deletes test index created for integration tests
+func (a *ODFECLISuite) DeleteEcommerceIndex() {
+	_, err := a.callRequest(http.MethodPut, HelperLoadBytes("ecommerce.json"), fmt.Sprintf("%s/%s/_doc", a.Profile.Endpoint, EcommerceIndexName))
```

Comment:
Instead of calling it DeleteEcommerceIndex, can we just say DeleteIndex and pass in the index name?

Reply:
Ack